tools/supervisor/datalab.conf

```diff
@@ -0,0 +1,10 @@
+[supervisord]
```

Comment:
I think this file should go into the containers directory.
Assumption is that this is only going to be used within the docker container, not outside.